webapp/src/main/webapp/vue-apps/CallButton/components/JitsiMeetButton.vue

```diff
@@ -48,6 +59,10 @@ export default {
     parentClasses: function() {
       return this.callSettings.context.parentClasses;
     },
+    displayTooltip: function() {
+      console.log('test');
```

Comment:
Could you remove this logging instruction ?

Reply:
done